src/app/server/CommissioningWindowManager.cpp

```diff
@@ -242,6 +242,49 @@ void CommissioningWindowManager::OnSessionEstablished(const SessionHandle & sess
     }
 }
 
+// Function called when NFC-based commissioning has been started.
```

Comment:
The documentation should be in the header, not here.

That said, it's really not clear when this should be called, so it's hard to evaluate whether the implementation is correct (or even whether the name is; presumably this is something to do with the "deferred" nature of NFC commissioning, and it might be better to name it accordingly, not with NFC-specific naming, especially if it's not inside NFC ifdefs).